### PR TITLE
feature/walkthrough

### PR DESCRIPTION
**Pull request by Jaspreet-Nijjar**
I have created the Walkthrough component which is a multi-step form following the Figma design. I have implemented all required functionality including moving between slides and added the previous functionality too. The progress bar is fully tested and also works.

### CODE REVIEW

**Review by lauraehiller — CHANGES_REQUESTED** [review]
Looks good, I'll approve once the merge conflicts are resolved!

**Comment by Jaspreet-Nijjar** [comment]
I've resolved the merge conflicts and created a new pull request to merge into the main branch.

**Review by lauraehiller — APPROVED** [review]
LGTM

**Pull request merged**
